ccpp/build_ccpp.sh

```diff
@@ -7,7 +7,7 @@ set -eu
 VALID_MACHINES=( wcoss_cray wcoss_dell_p3 gaea.intel jet.intel \
                  hera.intel orion.intel \
                  cheyenne.intel cheyenne.intel-impi cheyenne.gnu cheyenne.pgi endeavor.intel \
-                 stampede.intel supermuc_phase2.intel macosx.gnu \
+                 stampede.intel supermuc_phase2.intel comet macosx.gnu \
```

Comment:
GFSv16 does not run with ccpp. Since the code changes need to production branch, I suggest not to include any another changes related to ccpp.